client/my-sites/plan-features-2023-grid/_media-queries.scss

```diff
@@ -0,0 +1,137 @@
+// Variables
```

Comment:
NIT - curious, why do we underscore the file name `_media-queries.scss`? 🤔

Reply:
By default, Saas would transpile this into its own CSS file (`media-queries.css`). By adding the underscore, we specify that this is a Saas partial, and a CSS file will not be generated for this. Our build pipeline probably combines those CSS files later, but that's a different story. 😅

Reply:
ah yes, I'm getting old. Thanks for explaining :-)